commit/factory.go

```diff
@@ -2,19 +2,29 @@ package commit
 
 import (
 	"context"
+	"errors"
 	"math/big"
 
-	"github.com/smartcontractkit/chainlink-ccip/internal/reader"
-	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
-
 	"google.golang.org/grpc"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
 	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
-
 	"github.com/smartcontractkit/libocr/commontypes"
 	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
 	ocr2types "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
-	libocrtypes "github.com/smartcontractkit/libocr/ragep2p/types"
+	ragep2ptypes "github.com/smartcontractkit/libocr/ragep2p/types"
+
+	"github.com/smartcontractkit/chainlink-ccip/internal/reader"
+	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
+)
+
+const (
+	// defaultMsgScanBatchSize is the default batch size for sequence number range queries.
+	// Since 256 messages is the limit for a merkle root, this is set to 256.
+	// NOTE: maybe we can also set this in the OCR config, offchainConfig.
+	defaultMsgScanBatchSize = 256
```

Comment:
Looks like there's already a constant for this
```suggestion
	defaultMsgScanBatchSize = merklemulti.MaxNumberTreeLeaves
```

Reply:
Fixed, just used that const directly.